packages/cmf/src/selectors/components.md

```diff
@@ -0,0 +1,62 @@
+# components selectors
+
+## cmf.selectors.components.get
+
+How to use:
+
+```javascript
+import { selectors } from '@talend/react-cmf';
+
+const defaultValue = {};
+
+function mapStateToProps(state) {
+    return {
+        my: selectors.components.get(state, 'my.data', defaultValue);
+    }
+}
+```
+
+This selector accept the following arguments:
+
+| name | description |
+| -- | -- |
+| state| redux state |
+| path | path to collections which is stored in cmf.components (support dotted accessor like lodash get) |
+| default value | the default value (optional) |
+
+The selector works the same as selectors.collections.get, just it returns data that is stored in cmf.components,
+not in cmf.collections
+
+## cmf.selectors.components.toJSMemoized
+
+Unlike cmf.selectors.components.get, cmf.selectors.components.toJSMemoized gets collection from state,
```

Comment:
this is not a collection here but a component state.
We will have two kind of state. Hard to find a better name here.